src/core/components/odsCoreProvider/odsCoreProvider.mdx

```diff
@@ -0,0 +1,17 @@
+import { ArgTypes, Meta, Description, Title, Subtitle } from '@storybook/blocks';
+import { OdsCoreProvider } from './odsCoreProvider';
+
+<Meta title="Core/Components/OdsCoreProvider" />
+
+<Title>OdsCoreProvider</Title>
+
+The `<OdsCoreProvider />` is a React context that defines the image and link elements to be rendered across all the ODS
+components, defaulting such values to `<img />` and `<a />` tags. The provider is used by the `<AvatarBase />` and
+`<LinkBase />` components, which are in turn used by all the other core and module components.
+
```

Comment:
Maybe we could add a bit more implementation detail here with including snippet for Next setup: 

```suggestion
import { OdsCoreProvider } from '@aragon-ods';
import Image from 'next/image';
import Link from 'next/link';

const App = () => {
  return (
    <OdsCoreProvider values={{ Link: Link, Img: Image }}>
      {/* The rest of your app components go here */}
    </OdsCoreProvider>
  );
};

export default App;
```

Reply:
Yes makes sense 👍🏻